datadog_checks_dev/datadog_checks/dev/tooling/create.py

```diff
@@ -121,10 +121,16 @@ def create_template_files(template_name, new_root, config, read=False):
     for root, _, template_files in os.walk(template_root):
         for template_file in template_files:
             if not template_file.endswith(('.pyc', '.pyo')):
-                # Use a special README for the marketplace/partner support_type integrations
-                if template_file == 'README.md' and config.get('support_type') == 'partner':
-                    template_path = path_join(TEMPLATES_DIR, 'marketplace/', 'README.md')
-                    file_path = path_join("/", config.get('check_name'), "README.md")
+                if template_file == 'README.md' and config.get('support_type') in ('partner', 'contrib'):
```

Comment:
```suggestion
                if template_file == 'README.md':
```

Reply:
If we open the clause here, then core READMEs won't get processed in the `else` clause on line 142